client/src/components/DataEntry/BoardTable.tsx

```diff
@@ -8,10 +8,12 @@ import { BallotStatus } from '../../types'
 import LinkButton from '../Atoms/LinkButton'
 import { IBallot } from '../MultiJurisdictionAudit/RoundManagement/useBallots'
 import { IAuditBoard } from '../UserContext'
+import StatusTag from '../Atoms/StatusTag'
 
 const RightWrapper = styled.div`
   display: flex;
-  justify-content: flex-end;
+  align-items: flex-start;
```

Comment:
i think `center` or `baseline` would look better, so the text lines up within the tags/buttons